tests/python_package_test/test_sklearn.py

```diff
@@ -540,6 +540,17 @@ def test_metrics(self):
         self.assertIn('l2', gbm.evals_result_['training'])
         self.assertIn('mape', gbm.evals_result_['training'])
 
+        # non-default metric with multiple metrics in eval_metric for LGBMClassifier
+        X_classification, y_classification = load_breast_cancer(True)
+        params_classification = {'n_estimators': 2, 'verbose': -1, 'objective': 'binary', 'metric': 'binary_logloss'}
+        params_fit_classification = {'X': X_classification, 'y': y_classification,
+                      'eval_set': (X_classification, y_classification), 'verbose': False}
```

Comment:
Please fix one linting error:
```
./tests/python_package_test/test_sklearn.py:547:23: E128 continuation line under-indented for visual indent
```

Reply:
Fixed!

Reply:
Thank you! Sorry for the inconvenience, but could you please rebase to the latest `master`? I've just fixed R-package builds: #3225. Those failures prevents from merging.

Reply:
Rebased, hopefully I did it correctly as it is my first time rebasing an upstream branch 😄

Reply:
Thanks!
But seems that something went wrong: see number of changed files

![image](https://user-images.githubusercontent.com/25141164/87437773-66c4f800-c5f7-11ea-9adb-ef8b08173a14.png)

Maybe you can allow editing from maintainers and then I'll be able to help with cleaning up the rebasing?
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
<img width="274" alt="Screenshot 2020-07-14 at 22 34 14" src="https://user-images.githubusercontent.com/7747862/87438571-4b231700-c622-11ea-8abf-a58776c1a47a.png">
 editing for maintainers is enabled. Sorry for this.

Reply:
No problem! Seems I was able to exclude excess files.